src/trend/filterRow.js

```diff
@@ -77,29 +132,19 @@ function FilterRow(props) {
       keyComponent = null;
       cmpComponent = (
         <Field component={BootstrapField} name={`${name}.cmp`} type={"select"}>
-          {rangeCmp}
+          {dateRangeCmp}
         </Field>
       );
       valueComponent = (
         <>
           <Field
-            component={BootstrapDateField}
-            views={["year", "month", "date"]}
-            openTo={"month"}
-            format={"DD/MM/YYYY"}
-            name={`${name}.value.0`}
-            outputFormat={"YYYY-MM-DD"}
-            type={"date"}
-          />
-          &nbsp;
-          <Field
-            component={BootstrapDateField}
-            views={["year", "month", "date"]}
-            openTo={"month"}
-            format={"DD/MM/YYYY"}
-            outputFormat={"YYYY-MM-DD"}
-            name={`${name}.value.1`}
-            type={"date"}
+            component={DateRangeField}
+            // views={["year", "month", "date"]}
+            // openTo={"month"}
+            dateFormat={"dd/MM/yyyy"}
```

Comment:
why is the `dateFormat` here inverse of `filter.DATE_FORMAT`?

Reply:
Ah, good pickup. That's the format that the date is *displayed* as, whereas the format used for the actual communication/storage of the date is defined here: https://github.com/ewels/MegaQC/pull/121/files/7a3b97deab2c20d4a77ff82dd9740ccba713d4c1#diff-712e238a99b84f4ec3c1979f92de7296R15.

I like that we use ISO dates internally, but they're a bit opaque for users to actually see. However, there are big issues with `dd/MM/yyyy` as well, since it will look backwards to Americans. Perhaps in a later update I'll look for a locale-sensitive date format, which I believe this component does support.

Reply:
That's about what I thought. Personally I like using ISO everywhere because, aside from "hooray for standards", it triggers the "something unexpected, take a closer look at this" response in the user regardless of which month/day order they normally use. That's purely my POV/opinion though, so no reason to block the PR.

Locale-sensitive is also a good way to solve it, but agree not really necessary for initial work.

Reply:
Also, `mm/dd/yy` is clearly a ridiculous way to format dates and we should actively discourage it 😅

Reply:
If only it were that simple